specs/AppLifecycle/EnvironmentVariables/AppLifecycle Environment Variables.md

```diff
@@ -137,8 +138,10 @@ Notes:
 
 When uninstalling a package, environment cleanup and restoration is triggered.
 
-1. Any variables created by the app are deleted.
-2. Where a variable was not originally created by the app, but was last modified
+1. Any variables created by the app are deleted, unless another packaged app
+   has also modified that variable, in which case the previous value is restored
```

Comment:
Oh yes, that didn't make any sense. I removed that clause. Rather than going into detail in this summary, I'll leave the details in the table that covers all the cases.